mackerel/api.go

```diff
@@ -1,54 +1,34 @@
 package mackerel
 
 import (
-	"bytes"
-	"encoding/json"
 	"fmt"
 	"net/http"
-	"net/http/httputil"
-	"net/url"
 	"time"
 
-	"github.com/mackerelio/golib/logging"
 	mkr "github.com/mackerelio/mackerel-client-go"
 )
 
-var logger = logging.GetLogger("api")
-
 // API is the main interface of Mackerel API.
 type API struct {
-	BaseURL        *url.URL
-	APIKey         string
-	Verbose        bool
-	UA             string
-	DefaultHeaders http.Header
-}
-
-// Error represents API error
-type Error struct {
-	StatusCode int
-	Message    string
-}
-
-func (aperr *Error) Error() string {
-	return fmt.Sprintf("API error. status: %d, msg: %s", aperr.StatusCode, aperr.Message)
```

Comment:
This is different from ApiError of mackerel-client-go so log message can change but maybe ok...